server/container_update_resources_test.go

```diff
@@ -120,4 +118,36 @@ var _ = t.Describe("UpdateContainerResources", func() {
 			Expect(err).NotTo(BeNil())
 		})
 	})
+
+	t.Describe("UpdateContainerResources with nri enabled", func() {
+		// Prepare the sut
+		BeforeEach(func() {
+			beforeEach()
+			serverConfig.NRI.Enabled = true
```

Comment:
@Tal-or Wasn't your initial take on this to unconditionally run all unit tests with NRI enabled ? I think that could still be a good idea...

Reply:
I actually followed an earlier request from @haircommander.
But it's doable indeed

Reply:
Okay, sorry I missed that. Let's go with what  @haircommander think is the best course of action.